ui/src/logs/components/LogsTable.tsx

```diff
@@ -109,7 +107,7 @@ class LogsTable extends Component<Props, State> {
     }
 
     if (scrollToRow) {
-      scrollTop = calculateScrollTop(currentMessageWidth, data, scrollToRow)
+      scrollTop = calculateScrollTop(scrollToRow)
```

Comment:
We can probably delete the `calculateScrollTop` and just replace it here with `scrollToRow * ROW_HEIGHT`

Reply:
I think we should keep the function as it encapsulates the logic